components/scream/tests/rrtmgp/rrtmgp_stand_alone.cpp

```diff
@@ -173,7 +176,26 @@ namespace scream {
         auto d_rei = field_mgr.get_field("eff_radius_qi").get_reshaped_view<Real**>();
         auto d_cld = field_mgr.get_field("cldfrac_tot").get_reshaped_view<Real**>();
         auto d_mu0 = field_mgr.get_field("cos_zenith").get_reshaped_view<Real*>();
-        auto d_gas_vmr = field_mgr.get_field("gas_vmr").get_reshaped_view<Real***>();
+
+        auto d_qv  = field_mgr.get_field("qv").get_reshaped_view<Real**>();
```

Comment:
Seems like these should be accessed and populated in a loop similar to how they are accessed in `atmosphere_radation`? That way if the list of active gases changes for some reason this code stays up to date. Is there a reason to do it explicitly here?